google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java

```diff
@@ -90,21 +91,16 @@
   private static final TableId OTHER_TABLE_ID = TableId.of(PROJECT, DATASET, OTHER_TABLE);
   private static final TableId TABLE_ID_WITH_PROJECT = TableId.of(PROJECT, DATASET, TABLE);
   private static final Field FIELD_SCHEMA1 =
-      Field.newBuilder("StringField", LegacySQLTypeName.STRING)
+      Field.newBuilder("BooleanField", LegacySQLTypeName.BOOLEAN)
```

Comment:
I'm not sure what these test changes to the schema types have to do with the current PR. Just simplifying the schema?

Reply:
It does simplify the schema but mainly so that the schema and the table data match.